src/app/book/components/book/book.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+
+import { BookModel } from '../../../core/models/book';
+
+@Component({
+  selector: 'app-book',
+  templateUrl: './book.component.html',
+  styleUrls: ['./book.component.scss'],
```

Comment:
just for your understanding: we do not add styleUrls if don't have any component styles

Reply:
fixed